### DISCUSSION TITLE
Erro na instalação do Dokcer

### DISCUSSION THREAD

**Discussion by JotaPeCarvalho** [balta-io/forum#71] [Tira Dúvidas] [answered]
Não consigo intalar o docker, aparece a seguinte mensagem de erro: 
![image](https://user-images.githubusercontent.com/91575096/198898463-f0c71f24-5ae0-42ac-9686-8dc7e4e10e38.png)

**Comment by andrebaltieri** [comment, marked as answer]
@JotaPeCarvalho já tentou reiniciar o Windows?

**Reply by JotaPeCarvalho** [reply]
Já tentei de tudo, balta.

**Reply by andrebaltieri** [reply]
Você quer o Docker só para instalar o SQL Server? Se sim, não precisa... tem o Express para Windows
https://www.microsoft.com/pt-br/download/details.aspx?id=101064

**Comment by JotaPeCarvalho** [comment]
Beleza, eu continuei no express…

Em ter., 1 de nov. de 2022 às 14:26, Andre Baltieri <
***@***.***> escreveu:
